src/istio/mixerclient/check_cache_test.cc

```diff
@@ -49,16 +49,18 @@ class CheckCacheTest : public ::testing::Test {
     CheckResponse ok_response;
     ok_response.mutable_precondition()->set_valid_use_count(1000);
     // Just to calculate signature
-    EXPECT_ERROR_CODE(Code::NOT_FOUND, cache_->Check(attributes_, FakeTime(0)));
+    EXPECT_ERROR_CODE(Code::NOT_FOUND,
+                      cache_->Check(attributes_, FakeTime(0), nullptr));
     // set to the cache
```

Comment:
Please add a test case to test caching of route_directive.

Reply:
done